### ISSUE TITLE
Generate SQL for column with comment creates invalid DDL

### ISSUE THREAD

**Issue by elring** [dbeaver/dbeaver#11978]
#### System information: 
- OS: Linux Mint 20
- DBeaver version: 21.0.2 EA

#### Connection specification:
- Database: Postgresql 12

#### Describe the problem you're observing:
Function Generate SQL DDL for any column (with comment) creates invalid DDL - with semicolon inside comment.

#### Steps to reproduce
Use any table, for example 
```
CREATE TABLE public.testtable (
	id serial NOT NULL,
	testchar varchar NULL,
	CONSTRAINT testtable_pkey PRIMARY KEY (id)
);
COMMENT ON COLUMN public.testtable.testchar IS 'this is my comment';
```
Try to get DDL only for column testchar (right click on column, Generate SQL). The result SQL is:
```
ALTER TABLE public.testtable ADD testchar varchar NULL -- this is my comment;
COMMENT ON COLUMN public.testtable.testchar IS 'this is my comment';
```
Notice, that in ALTER TABLE statement, the semicolon is now part of the of the (useless) SQL -- comment. The whole 2 line SQL can't be simply executed, because the first SQL statement has no proper ending.

The semicolon should be before the -- comment:
```
ALTER TABLE public.testtable ADD testchar varchar NULL; -- this is my comment
COMMENT ON COLUMN public.testtable.testchar IS 'this is my comment';
```
or comment (on first statement) should be removed

```
ALTER TABLE public.testtable ADD testchar varchar NULL;
COMMENT ON COLUMN public.testtable.testchar IS 'this is my comment';
```

**Comment by LonwoLonwo** [comment]
Thanks for the bug report @elring 
We will fix it in next releases.

**Comment by LonwoLonwo** [comment]
Fixed in DBeaver 21.1.0
Will be available [here](https://dbeaver.io/) on 2021-05-31

**Comment by Matvey16** [comment]
Verified
